notes/points-markets-settlement.md

```diff
@@ -0,0 +1,103 @@
+
+`Inclusivity Threshold` (40.00%)
+: The percentage of issued points that must be eligible for the conversion
+
+`Liquidity Threshold` (5.00%)
+: The percentage of issued points that must be met by the bid depth (quantity of resting buy orders or points otherwise able to be sold) in the market within `valid slippage` (%) of the current market price.
+
+`Valid Slippage` (20.00%)
+: The range around the current market price at which available bids are deemed to constitute ‘liquidity’ for the purpose of an ***Airdrop Event*** in which points have become tradable.
+
+`Minimum Trading Window` (48 hours)
+: The minimum amount of cumulative time over which a market meeting the liquidity and slippage thresholds must exist to trigger an ***Airdrop Event***.
+
+`Early Termination Period` (24 hours)
+: The amount of time before a scheduled ***Airdrop Announcement***
+
+
+### Airdrop Event
+
+An ***Airdrop Event*** is deemed to have occurred when one or more of the following conditions is satisfied:
+
+1. At least `Inclusivity Threshold` (%) of all issued *points* have undergone *conversion* (defined above) since the inception of the points programme.
+
+2. It is no longer possible for *conversion* to happen, for example if the protocol ceases operation. (In this case the *mean conversion rate* will be zero.)
+
+3. Points become freely tradable by the majority of points holders to the extent that a current market price per point exists and the cumulative historic trading volume plus the available liquidity within `Valid Slippage` of the current market price is at least `Liquidity Threshold` (%) of the total number of issued points for a (not necessarily contiguous) period of no less than `Minimum Trading Window`.  
+
+Note: adjustments to points balances such as normal or reverse splits (1 point becomes, 2 points, 10 points, 100 points, 0.5 points, 0.1 points, etc.), multiplying some users points by some value, etc. etc. do not constitute an *Airdrop Event* and simply impact the market price of a point as an issuance of new points would.
+
+Once an ***Airdrop Event*** has occurred and all input to the ***Valuation Algorithm*** is available, the *mean conversion rate* is calculated and the market will be settled. 
+
+
+### Imminent Airdrop Event
+
+An ***Imminent Airdrop Event*** will occur following any official confirmation (including implied confirmation as a result of the action happening before any official communication) of a future *conversion*, listing, or other action that would constitute an ***Airdrop Event***. 
+
+For an announcement or action to trigger an ***Imminent Airdrop Event*** it must originate from or be endorsed as truthful and accurate by the project or another entity with sufficient control to initiate an ***Airdrop Event***.
+
+The *termination date/time* shall be `Early Termination Period` before any announcement in which the specifics of the ***Airdrop Event*** that are required by the ***Valuation Algorithm*** are confirmed.
```

Comment:
Is the default termination date always unspecified (market goes on into eternity if the airdrop event does not happen) or will markets have the option to specify that (and perhaps trigger a termination at `0` valuation then?). Might be worth clarifying that in this note.

Reply:
Add this in to the note and also run this past the market makers that have helped in creating this

Reply:
Added optional `Expiry Date/Time` to cover this case